data/templates.json

```diff
@@ -0,0 +1,170 @@
+[
+  {
+    "pid": "1",
+    "name": "Public template for documents",
+    "description": "Public template for documents.",
+    "organisation": {
+      "$ref": "https://ils.rero.ch/api/organisations/1"
+    },
+    "creator": {
+      "$ref": "https://ils.rero.ch/api/patrons/1"
+    },
+    "visibility": "public",
+    "template_type": "doc",
+    "data": {}
```

Comment:
For me it's a non sense to have templates without any data. At least the `$schema` corresponding to the template_type should be present. Maybe need to discuss with PO about this point ?

Reply:
I agree with @zannkukai and I have the same feeling for the titles of these templates, we should have something less generic, like "Document template for book" for instance. It will help to understand what theses templates are for.

Reply:
while waiting for the fixtures to come from Nicolas, I intentionally put nothing in the `data`. So this will be addressed in the next PR.